specs/centos_ci.adoc

```diff
@@ -2,28 +2,49 @@
 
 = CentOS CI
 
-CentOS CI should be configured to launch applicable set of tests as an
-acceptance tests for every new built rpm package.
+We need to make sure that new code doesn't break existing codebase and check
+that new features works as early as possible. This can be done by running
+acceptance integration tests automatically, using CentOS CI infrastructure.
 
 == Problem description
 
-None
+CentOS CI
+will be configured to launch acceptance integration test run for every new
+nightly rpm build from tendrl nightly copr project. As set of implemented
+features in tendrl codebase grows, the set of acceptance tests is extended to
+cover new features.
 
 == Use Cases
 
-* Acceptance testing for every new rpm package.
+* Acceptance testing for every new build from tendrl copr nightly project.
+* The same set of tests should be run for stable builds from current tendrl
+  copr as well.
 
 == Proposed change
 
-* CentOS CI host provisioning system have to be utilized for testing machines
+* New copr tendrl project with nightly builds needs to be created. Right now
+  we have just copr project for
+  https://copr.fedorainfracloud.org/coprs/tendrl/tendrl/[stable tendrl buils],
+  but for the purposes of CI proposed here, we need another one with the latest
+  code.
+
+* When nightly build fails, no acceptance tests are executed and the build
+  needs to be fixed first.
+
+* Acceptance tests executed by CentOS CI system will work with rpm builds only.
+
+* CentOS CI host provisioning system will be utilized for testing machines
   provisioning.
 
 * Installation and configuration of all Tendrl components should be automated
-  for usage in CI (this contains also installation and configuration of all
-  additional required components including gluster and ceph).
+  via ansible playbooks/roles from
+  https://github.com/Tendrl/usmqe-setup[usmqe-setup] repository for usage in CI
```

Comment:
> How the nightly packages are built

This is a question for the dev team I guess. From qe team point of view, I would be ok with any build process which is properly automated using either fedoraproject's copr or centos's koji so that it's possible to identify nighly rpm builds (via tags, generated yum repository) for each date.